lms/djangoapps/instructor/views/instructor_dashboard.py

```diff
@@ -250,6 +250,7 @@ def instructor_dashboard_2(request, course_id):
         'generate_bulk_certificate_exceptions_url': generate_bulk_certificate_exceptions_url,
         'certificate_exception_view_url': certificate_exception_view_url,
         'certificate_invalidation_view_url': certificate_invalidation_view_url,
+        'xqa_server': settings.FEATURES.get('XQA_SERVER', "http://your_xqa_server.com"),
```

Comment:
What exactly is `XQA_SERVER` setting achieving?

Reply:
`xqa_server` is used in `xqa_interface.html`  which is now included in `open_response_assessment.html` and it is used in Staff Debug for logging data.

Reply:
Sure, but how does it help in what we are trying to achieve?

Reply:
while setting up staff debug by `setup_debug`, `xqa_server` URL is required. for the context, you can view `setup_debug` function in [xqa_interface.html](https://github.com/edx/edx-platform/pull/22989/files#diff-cd445200ec32014decbe22ef2eca06f0L22) as on line 22 `sendlog` is called in which `xqa_server` is used.  It was added to the context on courseware page but by unknown reasons omitted from the instructors page.